conftest.py

```diff
@@ -1,9 +1,34 @@
-from importlib.util import find_spec
 import pkg_resources
 
+try:
+    from pytest_astropy_header.display import PYTEST_HEADER_MODULES, TESTED_VERSIONS
+    ASTROPY_HEADER = True
+except ImportError:
+    ASTROPY_HEADER = False
+
 entry_points = []
 for entry_point in pkg_resources.iter_entry_points('pytest11'):
     entry_points.append(entry_point.name)
 
-if "asdf_schema_tester" not in entry_points and find_spec('asdf') is not None:
```

Comment:
I don't think we need this since asdf is installed by `install_requires`? Unless this is doing some dark magic that I don't understand, @WilliamJamieson ?

Reply:
I believe this is redundant with any of the newer versions of asdf (2.8+).